appveyor.yml

```diff
@@ -61,7 +61,7 @@ environment:
     - JOB: "Build"
       GENERATOR: Ninja
       CONFIGURATION: "Release"
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2019
```

Comment:
Can we switch this one to MSVC 2015 instead? So that we ensure we're still compatible with it.